sky/resources.py

```diff
@@ -172,7 +177,11 @@ def __init__(
         self._try_validate_disk_tier()
         self._try_validate_ports()
 
-    @service_catalog.use_default_catalog_if_failed
+    # When querying the accelerators for the instance type, we will check the
```

Comment:
Sorry, not following. Which callsite inside repr causes this? I only see some `{accelerators}` formatting.

Reply:
Got it, tried to incorporate this in.